file_upload_handler/src/server.js

```diff
@@ -0,0 +1,67 @@
+import express from 'express'
+import multer from 'multer'
+
+const app =express();
```

Comment:
```
const app = express();
```

Reply:
If you're using VSCode, install the Prettier Code Formatter addon and run it on this file